### PR TITLE
[10] remove jquery

### LINKED ISSUE #10: Remove jQuery dependency
A suggestion for this release: is it feasible to remove the jQuery dependency here? I think that most new sites using this won't have jQuery already enqueued, so adding the extra 30k to enqueue the library will hurt performance.

It looks like jQ is mainly used for adding click event listeners and adding/removing classes, and all modern browsers should support DOM functionality like `addEventListener` and `classList`.

### PR DESCRIPTION
**Pull request by jazzsequence**
Refactors the `main.js` file to remove the jQuery.
Tested locally to make sure everything still works.

Some changes from #13 got included so removed those, too, to make sure this branch is clean.

fixes #10

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -1,107 +1,113 @@
...
-	if ( cookiePrefsButton ) {
-		cookiePrefsButton.on( 'click', toggleCookiePrefs );
-		applyCookiePrefs.on( 'click', updateConsentCategories );
-	}
+/**
+ * Show or hide the cookie preferences.
+ */
+function toggleCookiePrefs() {
+	const allowAllClasses = giveConsentButton.classList,
+		allowFunctionalClasses = revokeConsentButton.classList;
```
Comment (roborourke) [inline assets/js/main.js:87]: Need spaces for alignment here

**Review by roborourke — APPROVED** [review]
Very minor code style thing but it's not a blocker really.

One thing we might want to address in another issue is that the class names are generic and not scoped eg. using BEM naming or an `altis` prefix or something like that. I think most of these would be fine but we should take extra care.

File: assets/js/main.js (hunk 1 above)
Comment (jazzsequence) [reply assets/js/main.js:87]: I've been on projects where these are aligned and others where they're not so I was `¯\_(ツ)_/¯` _guess I'll leave them_ even if they look wrong to me, too.

**Comment by jazzsequence** [comment]
> One thing we might want to address in another issue is that the class names are generic and not scoped eg. using BEM naming or an altis prefix or something like that. I think most of these would be fine but we should take extra care.

Yeah, I'm not attached to any of the class names and agree that they're very generic (that was the idea). Happy to open a ticket for a consistent naming structure.

**Pull request merged**